packages/backend/src/apps/formsg/auth/index.ts

```diff
@@ -20,8 +20,7 @@ const auth: IUserAddedConnectionAuth = {
       required: true,
       readOnly: false,
       value: null,
-      description:
-        'Enter your Form URL e.g. https://form.gov.sg/654ab1234abc1a012345f1e0b',
+      description: 'Click share on your form and copy the link from there',
```

Comment:
```suggestion
      description: 'Click share on your form and copy the link',
```
i still think it's alr super clear the have the example URL there. no one has messed this part uup